ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -836,8 +847,8 @@ tests:
     observers:
       enable:
       - observers-resource-watch
-    workflow: network-flow-matrix-bm
-- as: metal-ovn-network-flow-matrix-single-node-bm
+    workflow: telco5g-network-flow-matrix-bm
```

Comment:
Are these workflows and jobs purposely tagged `telco5g` now?

Reply:
Yes